docs/book/src/topics/language-specific-examples/azure-identity-sdk.md

```diff
@@ -1,33 +1,3 @@
-| C#                    | [azure-sdk-for-net](https://github.com/Azure/azure-sdk-for-net)       | [Azure.Identity_1.9.0](https://github.com/Azure/azure-sdk-for-net/releases/tag/Azure.Identity_1.9.0)      | [Azure.Identity_1.5.0](https://github.com/Azure/azure-sdk-for-net/releases/tag/Azure.Identity_1.5.0)    |
-| JavaScript/TypeScript | [azure-sdk-for-js](https://github.com/Azure/azure-sdk-for-js)         | [@azure/identity_3.2.0](https://github.com/Azure/azure-sdk-for-js/releases/tag/@azure/identity_3.2.0)     | [@azure/identity_2.0.0](https://github.com/Azure/azure-sdk-for-js/releases/tag/@azure/identity_2.0.0)   |
-| Python                | [azure-sdk-for-python](https://github.com/Azure/azure-sdk-for-python) | [azure-identity_1.13.0](https://github.com/Azure/azure-sdk-for-python/releases/tag/azure-identity_1.13.0) | [azure-identity_1.7.0](https://github.com/Azure/azure-sdk-for-python/releases/tag/azure-identity_1.7.0) |
-| Java                  | [azure-sdk-for-java](https://github.com/Azure/azure-sdk-for-java)     | [azure-identity_1.9.0](https://github.com/Azure/azure-sdk-for-java/releases/tag/azure-identity_1.9.0)     | [azure-identity_1.4.0](https://github.com/Azure/azure-sdk-for-java/releases/tag/azure-identity_1.4.0)   |
-
-## Examples
-
-### Using `DefaultAzureCredential`
-
-| Language              | Library                                                               | Example                                                                                           |
-| --------------------- | --------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
-| Go                    | [azure-sdk-for-go](https://github.com/Azure/azure-sdk-for-go)         | [Link](https://github.com/Azure/azure-workload-identity/tree/main/examples/azure-identity/go)     |
-| Python                | [azure-sdk-for-python](https://github.com/Azure/azure-sdk-for-python) | [Link](https://github.com/Azure/azure-workload-identity/tree/main/examples/azure-identity/python) |
-| JavaScript/TypeScript | [azure-sdk-for-js](https://github.com/Azure/azure-sdk-for-js)         | [Link](https://github.com/Azure/azure-workload-identity/tree/main/examples/azure-identity/node)   |
-| C#                    | [azure-sdk-for-net](https://github.com/Azure/azure-sdk-for-net)       | [Link](https://github.com/Azure/azure-workload-identity/tree/main/examples/azure-identity/dotnet) |
-| Java                  | [azure-sdk-for-java](https://github.com/Azure/azure-sdk-for-java)     | [Link](https://github.com/Azure/azure-workload-identity/tree/main/examples/azure-identity/java)   |
```

Comment:
IMO the examples section shouldn't be removed from this doc as it refers to examples in this GitHub repo

Reply:
> What if we deleted the code samples from this repo and embedded them in the conceptual doc instead? An easy way to do that is language-specific tabs. I'll even volunteer to open that docs PR. Let me know your thoughts.

I like this idea! I can help out with this change.

Reply:
The doc update is live at https://learn.microsoft.com/azure/aks/workload-identity-overview?tabs=dotnet#azure-identity-client-libraries.